drivers/modem/wncm14a2a.c

```diff
@@ -960,10 +981,12 @@ static void on_cmd_sockdataind(struct net_buf **buf, u16_t len)
 
 static void on_cmd_socknotifyev(struct net_buf **buf, u16_t len)
 {
-	char value[40];
+	char value[41];
```

Comment:
Why changing the length of the buffer here, was there an error in the original code?
There was a similar case in on_cmd_sockwrite(), on_cmd_sockdial() and on_cmd_sockexterror() where you did not increase the size of the value[] buffer.

Reply:
> Why changing the length of the buffer here, was there an error in the original code?

I have suspicions regarding the original code. I didn't change other buffers, because there're clearly destined for numeric data. Even if there was a thinko in the original code, and we'll go from 8 decimal digits to 7, it's a difference between (almost) 10 millions and (almost) 100 millions, and hopefully length/id's encoded there won't hit those values (which would be truncated). And going from char[8] to char[9] could increase stack usage considerably (e.g. 2x) for those functions.

Here however we have the whole command line, and I don't want to risk it, and go conservative with allowing (still) 40 actual chars.

All in all, the original author of this code, @mike-scott, is in reviewers, and certainly will correct me if anything.

Reply:
40 size is ok here.  Max incoming value would be:
"LTIME",yy/mm/dd,hh:mm:ss±zz
For a total of 31 bytes (including null)

Reply:
> 40 size is ok here.

Reverted back to 40.